docs/README.md

```diff
@@ -326,6 +327,17 @@ There are two methods for it:
 
 <br/>
 
+## 7. Difference between Attribute and Property
+- Attributes provide additional information about the HTML elements, attributes are always specified in the start tag and they usually come in name/value pairs.
+- Properties are the values associated with a JavaScript object, these objects are collection of unordered properties and these properties can be usually be changed, added, and deleted.
```

Comment:
Properties are accessed from DOM (Document Object Model) nodes.

Could you add this point also

Reply:
👍🏻